lib/cucumber/core/test/case.rb

```diff
@@ -53,10 +53,20 @@ def match_name?(name_regexp)
 
         def match_locations?(queried_locations)
           queried_locations.any? do |queried_location|
-            queried_location.match? location
+            matching_locations.any? do |location|
+              queried_location.match? location
+            end
           end
         end
 
+        def matching_locations
+          [
+            location,
```

Comment:
For my benefit as the naming is ambiguous, where is this defined? Github gives me a few opts.

Reply:
Its just a value attr, and it's set here: https://github.com/cucumber/cucumber-ruby-core/blob/baf204008e98900a04a1fa10dbabcc5a90c22c04/lib/cucumber/core/compiler.rb#L42

In more detail, `C::C::Test::Case#location` is a `Location::Precise` value of the Scenario line, set by `C::C::Compiler` when it creates the Test Case upon receiving the relevant information from the pickles compiler. Does this answer your question?